storage/src/main/scala/coop/rchain/storage/internal.scala

```diff
@@ -1,12 +1,10 @@
 package coop.rchain.storage
 
-private[storage] final case class Datum[A](a: A, persist: Boolean)
+final case class Datum[A](a: A, persist: Boolean)
 
 private[storage] final case class DataCandidate[C, A](channel: C, datum: Datum[A], datumIndex: Int)
 
-private[storage] final case class WaitingContinuation[P, K](patterns: List[P],
-                                                            continuation: K,
-                                                            persist: Boolean)
+final case class WaitingContinuation[P, K](patterns: List[P], continuation: K, persist: Boolean)
```

Comment:
Note that I've discussed with @henrytill and we are going to expose these two case classes until we come up with a better solution.